docs/azure_jumpstart_arcbox/ITPro/_index.md

```diff
@@ -446,6 +446,50 @@ After deployment is complete, its time to start exploring ArcBox. Most interacti
 
   ![Screenshot showing ArcBox Client VM with Hyper-V](./hypervterminal.png)
 
+- Alternately, you can use Azure CLI to [connect to Azure Arc-enabled servers using SSH](https://learn.microsoft.com/azure/azure-arc/servers/ssh-arc-overview?tabs=azure-cli).
+
+  - Windows (PowerShell)
+
+    ```powershell
+    az login
+
+    $rgName = "arcbox-rg"
+    $serverName = "ArcBox-Win2K22"
+    $localUser = "arcdemo"
+
+    az ssh arc --resource-group $rgName --name $serverName --local-user $localUser
+    ```
+
+  - Linux (shell)
+
+    ```shell
+    az login
+
+    export rgName="arcbox-rg"
+    export serverName="ArcBox-Win2K22"
+    export localUser="arcdemo"
+
+    az ssh arc --resource-group $rgName --name $serverName --local-user $localUser
+    ```
+
+    ![Screenshot showing usage of Remote Desktop tunnelled via SSH](./ssh_via_az_cli_01.png)
+
+    ![Screenshot showing usage of Remote Desktop tunnelled via SSH](./ssh_via_az_cli_02.png)
+
+- Following to the previous method, you can also use Azure CLI to connect to Azure Arc-enabled servers (Windows) using Remote Desktop tunnelled via SSH.
+
+  ```powershell
+  az login
+
+  $rgName = "arcbox-rg"
+  $serverName = "ArcBox-Win2K22"
+  $localUser = "arcdemo"
+
+  az ssh arc --resource-group $rgName --name $serverName --local-user $localUser --rdp
+  ```
```

Comment:
```suggestion
- Following the previous method, you can also use Azure CLI to connect to one of the Azure Arc-enabled servers, Hyper-V Windows Server virtual machines using Remote Desktop tunneled via SSH.

  ```powershell
  az login

  $serverName = "ArcBox-Win2K22"
  $localUser = "Administrator"

  az ssh arc --resource-group $Env:resourceGroup --name $serverName --local-user $localUser --rdp
  ```
```

Reply:
- Added explicitly to the lingo around Windows Server VMs

- Same comment on the _az login_ with SPN env vars.

-  Same comment on the $Env:resourceGroup env var.

- Local user updated to _Administrator_.

- Trying to RDP, fresh deployment, and getting this. Any thoughts? 
![image](https://user-images.githubusercontent.com/46730876/213961954-b4139244-6753-4afe-a37c-8e7ecf1f48ee.png)

Reply:
> Trying to RDP, fresh deployment, and getting this. Any thoughts?

Yes, I believe this is due to OpenSSH server not being installed on the Windows VMs. This is added in my branch, but I assume you deployed from the main branch which does not have the commands which installs this?

Since SSH also works for Windows, should we show that as well? Maybe before the _--rdp_ example?